packages/ndla-ui/src/NDLAFilm/FilmSlideshow.tsx

```diff
@@ -0,0 +1,264 @@
+/**
+ * Copyright (c) 2016-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+import { SwipeEventData, useSwipeable } from 'react-swipeable';
+import BEMHelper from 'react-bem-helper';
+import SafeLink from '@ndla/safelink';
+import { OneColumn } from '../Layout';
+import Spinner from '../Spinner';
+import { NDLAMovie } from './interfaces';
+import NavigationArrow from './NavigationArrow';
+import SlideshowIndicator from './SlideshowIndicator';
+
+interface Props {
+  autoSlide: boolean;
+  randomStart: boolean;
+  slideshow: Array<NDLAMovie>;
+  slideInterval: number;
+}
+
+const classes = new BEMHelper({
+  name: 'film-slideshow',
+  prefix: 'c-',
+});
+
+const defaultTransitionSwipeEnd = 'transform 600ms cubic-bezier(0, 0.76, 0.09, 1)';
+const defaultTransitionText = 'opacity 600ms ease';
+
+const renderSlideItem = (slide: any) => (
```

Comment:
Her veit vi jo kva type dette er så det er bedre å være eksplisitt.
```suggestion
const renderSlideItem = (slide: NDLAMovie) => (
```